apps/www/components/ui/accordion.tsx

```diff
@@ -28,7 +28,7 @@ const AccordionTrigger = React.forwardRef<
     <AccordionPrimitive.Trigger
       ref={ref}
       className={cn(
-        "flex flex-1 items-center justify-between py-4 font-medium transition-all hover:underline [&[data-state=open]>svg]:rotate-180",
+        "flex flex-1 items-center justify-between rounded-md py-4 font-medium transition-all hover:underline focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-4 [&[data-state=open]>svg]:rotate-180",
```

Comment:
@shadcn, few remarks:
- `"ring-offset-background"` is missing for general theming support (e.g. dark mode)
- I'd go with `"ring-offset-2"` to fit other components like `Button`
- Make it `my-2 py-2` instead of `py-4` for the focus ring to be closer to the button. That way, it won't overlap with the `AccordionContent` but stick with the same spacing
- `"disabled:opacity-50 disabled:pointer-events-none` for disabled state can be included (copied from `Button` as well)

Reply:
Also, it could be possible to add `"-mx-2 px-2"` to make the padding of the focus state a little bit spacier on x-padding. A bit hacky but looks better.

Reply:
Hi, is this pr still relevant? I wonder why it has not been merged yet.